sass/pages/_home.scss

```diff
@@ -1 +1,99 @@
-//This styles are specific styles for home.php
\ No newline at end of file
+//This styles are specific styles for home.php
+.latest_post{
+
+  @include pink_background_gradient;
+
+    .latest_post_wrapper{
+      padding: 3rem 1rem 2rem 1rem;
+
+      .latest_post_image{
+        @include image_box_shadow;
+      }
+    }
+  }
+
+  .post_content{
+    padding-top: 3rem;
+    padding-bottom: 1rem;
+  }
+
+  .all_posts{
+    background: linear-gradient(to right, $palepink_color, 5%, white);
+    padding-top: 2rem; 
+    padding-bottom: 2rem;
+
+    .blog_post{
+      margin: 1rem 1rem 2rem 1rem;
+    }
+  }
+
+  //Overlay effect on post images
+  
+  .overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: auto;
+    width: 100%;
+    opacity: 0;
+    transition: .5s ease;
+    background-color: $darkblue_color;
+ 
+ 
+  .container:hover & {
+    opacity: 0.9;
+  }
+ 
+ }
+  
+  // Text for overlay effect 
+   .text {
```

Comment:
Think what I'm trying to say here is: This class have a very generic name. I have no idea what text this is.
If it's connected to the overlay effect ->if it can't be without it, if it can't be reused anywhere else without the overlay thing. Then it has to be nested inside the overlay class so it's crystal-clear what text we are taking about.

If this text on the other hand have nothing to do with the overlay-effect and can act as a stand-alone component it should be included inside the typograpy.scss file insted. And have a name that goes with the "rules" for how classes are named in typography. 

I think nothing ever should be named to things like; create, image, text, wrapper or anything like that. The typography-file is an exeption as it's made to have control of ALL the text-styles. And have generic classes to be used as stand-alone components anywhere you want. (And therefore should the amount of allowed classes in that document be strictly limited)